internal/btf/types.go

```diff
@@ -1011,9 +1004,6 @@ func newEssentialName(name string) essentialName {
 }
 
 // UnderlyingType skips qualifiers and Typedefs.
```

Comment:
I could add an example for Copy which uses UnderlyingType?

Reply:
Something like this, though it would benefit from #633 for better output:

```Go
// Copy can be used with UnderlyingType to strip qualifiers from a type graph.
func ExampleCopy_stripQualifiers() {
	a := &Volatile{Type: &Pointer{Target: &Typedef{Name: "foo", Type: &Int{Size: 2}}}}
	b := Copy(a, UnderlyingType)
	// b has Volatile and Typedef removed.
	fmt.Printf("%4v", b)
	// Output:
}
```

Reply:
Looks good!